main_test.go

```diff
@@ -62,13 +25,13 @@ func TestComponent(t *testing.T) {
 		var opts = godog.Options{
 			Output: colors.Colored(os.Stdout),
 			Format: "pretty",
-			Paths:  flag.Args(),
+			Paths:  []string{"features/census", "features/homepage"},
```

Comment:
So we need to setup another godog.TestSuite to cover features in `features/censusEnabledGetDataCard`

Reply:
This will require you to create a different InitializeScenario function, maybe something like...

```
func InitializeScenarioWithGetDataCardEnabled(ctx *godog.ScenarioContext) {
	goCtx := context.Background()
	component, _ := feature.New(goCtx)
	component.Config.EnableGetDataCard = true // Update configuration for enabling changes to "Get data card"

	url := fmt.Sprintf("http://%s%s", component.Config.SiteDomain, component.Config.BindAddr)
	uiFeature := componenttest.NewUIFeature(url)

	RegisterSteps(ctx, component, uiFeature)
}
```

...in features/steps/steps.go